timApp/static/scripts/tim/plugin/imagex.ts

```diff
@@ -1248,8 +1252,14 @@ interface IAnswerResponse {
             <div class="pluginError" *ngIf="imageLoadError" [textContent]="imageLoadError"></div>
             <h4 *ngIf="header" [innerHtml]="header"></h4>
             <p *ngIf="stem" class="stem" [innerHtml]="stem"></p>
-            <div class="imagex-content">
+            <div class="imagex-content" style="position:relative; height: {{canvasheight}}px">
```

Comment:
```suggestion
            <div class="imagex-content" [style.height.px]="canvasheight" style="position: relative;">
```

Tosin ehkä laittaisin `position: relative`:n suoraan `imagex-content`-luokan tyyliksi.